### ISSUE TITLE
No url button (webui 2.5.1)

### ISSUE THREAD

**Issue by ylascaux** [shinken-monitoring/mod-webui#573]
Hi,

On the new version (2.5.1), I don't find the url button on top right of service page.
Are you remove it? Or maybe move to another place?

Thanks in advance.

**Comment by maethor** [comment]
Hi @hellnight 

I'm sorry, I'm not sure to understand what button you are talking about. Do you mean the note url?

Indeed I removed this button it in the last version. I thought nobody was using it. Was you?

**Comment by ylascaux** [comment]
Hi,
Yes It's the note url. I use it to link the doc for sysadmin :)

**Comment by maethor** [comment]
Right…

So, indeed, I removed this button. Sorry :(

I will try to make it appear in the Information tab as soon as possible. That would be ok for you ?

**Comment by ylascaux** [comment]
Yes it's ok for me. I just need this button :)
Thank you in advance.

**Comment by maethor** [comment]
@hellnight can you paste an exemple of this "notes_url" field please? I have a huge conflict between the shinken documentation and the last webui implementation, I want to know how you use this field in your config.

Is it like in http://shinken.readthedocs.io/en/latest/08_configobjects/hostextinfo.html:

    notes_url  http://url

Or are you using a more complicated syntax, like multiple urls or even icon, title, descriptions…? I found some crazy stuff in the old code.

Also, do you use `notes` and `action_url`?

**Comment by ylascaux** [comment]
@maethor I use just a simple action_url : 
An example of my config : 
```
##################################################
# SERVICES Inode 
##################################################
define service{
      host_name                 *
      use                                1h-oncall
      service_description       Inode $KEY$
      check_command             inode!$VALUE1$!$VALUE2$!$KEY$
      contact_groups            admins
      business_impact           2
      notification_options      w,u,c,r
      notifications_enabled     0
      action_url                     https://our_wiki/supervision/inodes
      duplicate_foreach         _inode
      register			0
}
```

**Comment by maethor** [comment]
Thank you.

@hellnight I just released 2.5.2 which reintroduces action_url and note_url buttons in the eltview information tab. It will take some time to be available in shinken.io, but you should see it tomorrow.

Please let me know if it's OK for you.

**Comment by ylascaux** [comment]
@maethor Thank you, it's work :)
